src/module.ts

```diff
@@ -72,7 +90,12 @@ export class Module<
   }
 
   context(store: Store<any>): Context<this> {
-    return new Context(createLazyContextPosition(this), store)
+    return new Context(
+      createLazyContextPosition(this),
+      store,
+      this.mutationNames,
+      this.actionNames
```

Comment:
Let's just pass `mutations` and `actions` here and move the process to collect `mutationNames` and `actionNames` into `Context`. So that we can only execute this calculation when the users use this feature.

The current implementation always collect `mutationNames` and `actionNames` when context will be created which is unnecessary overhead if the users don't use it.